Cargo.toml

```diff
@@ -12,21 +12,19 @@ keywords = ["gamedev", "ecs", "entity", "component"]
 categories = ["concurrency", "game-engines"]
 license = "MIT OR Apache-2.0"
 authors = ["slide-rs hackers"]
-exclude = ["bors.toml", ".travis.yml"]
-edition = "2018"
+include = ["src", "README.md", "LICENSE-MIT", "LICENSE-APACHE"]
+edition = "2021"
+rust-version = "1.60.0"
```

Comment:
the feature `nightly` was added some time back to introduce GAT and not increase the minimal rust version. we should check if its possible to just enable this by default now and if the required features are stabelized in 1.60.0

Reply:
I tried compiling with the `nightly` feature on stable, the necessary language features are not yet stabilized. I remember it's been forever that GAT are "about to be stabilized", but definitely worth keeping an eye out :)